src/main/kotlin/com/cognifide/gradle/aem/api/AemConfig.kt

```diff
@@ -123,7 +123,7 @@ class AemConfig(
      * Repeat install when failed (brute-forcing).
      */
     @Input
-    var installRetryTimes: Int = propParser.int("aem.install.retry.times", 3)
+    var installRetryTimes: Int = propParser.int("aem.install.retry.times", 1)
```

Comment:
this value was too big while assuming happy path (if oak:constraint will cause fail there will be no need to retry more); for edge cases still this value could be manually increased